exporter/signalfxexporter/config_test.go

```diff
@@ -21,15 +21,18 @@ import (
 	"testing"
 	"time"
 
+	"github.com/signalfx/signalfx-agent/pkg/apm/correlations"
```

Comment:
Can we give this an alias? `correlations` is easy to confuse with `correlation` in code later.